### PR TITLE
Improve AWS in arch and install

### PR DESCRIPTION
**Pull request by stg-0**

### CODE REVIEW

**Comment by stratiocommit** [comment]
This PR has modified files that have not been added for all versions. Please add them including the text [to be translated] so that the reviewer can easily upload or modify the translated versions. Missing folder/files:
EN - modules/ROOT/pages/architecture.adoc
EN - modules/ROOT/pages/installation.adoc

File: stratio-docs/es/modules/ROOT/pages/architecture.adoc [hunk 1]
```diff
@@ -49,7 +49,24 @@ En un *despliegue por defecto* se crean los siguientes objetos en cada proveedor
...
+* [silver]#6 _subnets_ con sus respectivas tablas de rutas.#
+** [silver]#3 _subnets_ públicas (una por AZ).#
+** [silver]#3 _subnets_ privadas (también una por AZ).#
+* [silver]#1 _NAT gateway_ por cada _subnet_ pública.#
+* [silver]#1 _Internet gateway_ para la VPC.#
+* [silver]#1 ruta por defecto en la tabla de rutas de cada _subnet_ privada para salir a internet a través de los _NAT gateways_.#
+* [silver]#1 ruta por defecto en la tabla de rutas de cada _subnet_ pública para salir a internet a través del _Internet Gateway_.#
+* 1 política de IAM para los nodos del _cluster_ (_nodes.cluster-api-provider-aws.sigs.k8s.io_).
+* 1 rol de IAM para los nodos del _cluster_ (_nodes.cluster-api-provider-aws.sigs.k8s.io_).
+* 2 _Security Groups_ para los workers y 3 para el _control-plane_.
```
Comment (tperez-stratio) [inline stratio-docs/es/modules/ROOT/pages/architecture.adoc:65]:
```suggestion
* 2 _Security Groups_ para los _workers_ y 3 para el _control-plane_.
```

File: stratio-docs/es/modules/ROOT/pages/architecture.adoc [hunk 2]
```diff
@@ -49,7 +49,24 @@ En un *despliegue por defecto* se crean los siguientes objetos en cada proveedor
...
+* VMs para _Workers_ (según descriptor del _cluster_ y autoescalado).
```
Comment (tperez-stratio) [inline stratio-docs/es/modules/ROOT/pages/architecture.adoc:66]:
```suggestion
* Máquinas virtuales para _Workers_ (según descriptor del _cluster_ y autoescalado).
```

File: stratio-docs/es/modules/ROOT/pages/architecture.adoc [hunk 3]
```diff
@@ -49,7 +49,24 @@ En un *despliegue por defecto* se crean los siguientes objetos en cada proveedor
...
+** 1 Volumen EBS para cada volumen persistente.
```
Comment (tperez-stratio) [inline stratio-docs/es/modules/ROOT/pages/architecture.adoc:67]:
```suggestion
** 1 volumen EBS para cada volumen persistente.
```

File: stratio-docs/es/modules/ROOT/pages/architecture.adoc [hunk 4]
```diff
@@ -49,7 +49,24 @@ En un *despliegue por defecto* se crean los siguientes objetos en cada proveedor
...
+* 1 Balanceador de carga tipo _Classic_ para la exposición de _Services_ tipo Load Balancer.
```
Comment (tperez-stratio) [inline stratio-docs/es/modules/ROOT/pages/architecture.adoc:68]:
```suggestion
* 1 balanceador de carga tipo _Classic_ para la exposición de _Services_ tipo _Load Balancer_.
```

File: stratio-docs/en/modules/ROOT/pages/architecture.adoc [hunk 5]
```diff
@@ -50,7 +49,24 @@ In a *default deployment*, the following objects are created in each cloud provi
...
+
+=== AWS unmanaged
+
+* [silver]#1 VPC.#
+* [silver]#6 subnets with their respective routing tables.#
+** [silver]#3 public subnets (one per AZ).#
+** [silver]#3 private subnets (also one per AZ).#
+* [silver]#1 NAT gateway for each public subnet.#
+* [silver]#1 Internet gateway for the VPC.#
+* [silver]#1 default route in the routing table of each public subnet to exit to the Internet through the NAT gateways.#
```
Comment (tperez-stratio) [inline stratio-docs/en/modules/ROOT/pages/architecture.adoc:61]:
```suggestion
* [silver]#1 default route in the routing table of each private subnet to exit to the Internet through the NAT gateways.#
```

File: stratio-docs/en/modules/ROOT/pages/architecture.adoc [hunk 6]
```diff
@@ -50,7 +49,24 @@ In a *default deployment*, the following objects are created in each cloud provi
...
+* [silver]#1 default route in the routing table of each public subnet to exit to the Internet through the Internet Gateway.#
+* 1 IAM policy for the cluster nodes (_nodes.cluster-api-provider-aws.sigs.k8s.io_).
+* 1 IAM role for cluster nodes (_nodes.cluster-api-provider-aws.sigs.k8s.io_).
+
```
Comment (tperez-stratio) [inline stratio-docs/en/modules/ROOT/pages/architecture.adoc:65]: En la versión en español está esta línea que aquí no aparece:
```suggestion
* 2 _Security Groups_ for the _workers_ and 3 for the _control-plane_.
```

File: stratio-docs/en/modules/ROOT/pages/architecture.adoc [hunk 7]
```diff
@@ -50,7 +49,24 @@ In a *default deployment*, the following objects are created in each cloud provi
...
+* VMs for _workers_ (according to cluster descriptor and auto-scaling).
+** 1 EBS volume for each persistent volume.
+* 1 network type load balancer for the exposure of load balancer type _Services_.
```
Comment (tperez-stratio) [inline stratio-docs/en/modules/ROOT/pages/architecture.adoc:68]:
```suggestion
* 1 load balancer type _Classic_ for the exposure of _Services_ type _Load Balancer_.
```

File: stratio-docs/en/modules/ROOT/pages/architecture.adoc [hunk 8]
```diff
@@ -50,7 +49,24 @@ In a *default deployment*, the following objects are created in each cloud provi
...
+** 1 listener per port for each service.
```
Comment (tperez-stratio) [inline stratio-docs/en/modules/ROOT/pages/architecture.adoc:69]:
```suggestion
** 1 listener per port for each _Service_.
```

File: stratio-docs/en/modules/ROOT/pages/architecture.adoc [hunk 9]
```diff
@@ -223,7 +246,7 @@ NOTE: The secondary CIDR assigned to the VPC for the pods must be indicated in t
 
 The CIDR of each subnet (obtained from the secondary CIDR of the VPC), must be the same as described above (with 18-bit mask), and the 3 subnets for pods must have the following tag: _sigs.k8s.io/cluster-api-provider-aws/association=secondary_.
 
-==== GCP and unmanaged Azure
+==== GCP and unmanaged AWS/Azure
```
Comment (tperez-stratio) [inline stratio-docs/en/modules/ROOT/pages/architecture.adoc:249]:
```suggestion
==== GCP and AWS/Azure unmanaged
```

File: stratio-docs/en/modules/ROOT/pages/architecture.adoc [hunk 10]
```diff
@@ -294,9 +317,13 @@ During the creation of the EKS cluster, an endpoint is created for the API Serve
 
 This endpoint is published to the internet, and its access is restricted with a combination of Amazon's Identity and Access Management (IAM) rules, and Kubernetes' native Role Based Access Control (RBAC).
 
+==== AWS unmanaged
+
+For API Server exposure, a load balancer is created with name `<cluster_id>-apiserver` and port 6443 accessible by the public network (the assigned public IP is the same as configured in the _Kubeconfig_) and one _Target group_ with the corresponding _control-plane_ nodes.
```
Comment (tperez-stratio) [inline stratio-docs/en/modules/ROOT/pages/architecture.adoc:322]:
```suggestion
For access to the API Server, a load balancer is created with name `<cluster_id>-apiserver` and port 6443 accessible by the public network (the assigned public IP is the same that resolves the URL of the _Kubeconfig_) and a _Target group_ with the nodes of the corresponding _control-plane_.
```

File: stratio-docs/en/modules/ROOT/pages/architecture.adoc (hunk 6 above)
Comment (iamjanr) [reply stratio-docs/en/modules/ROOT/pages/architecture.adoc:65]: Sip

**Review by iamjanr — APPROVED** [review]
LGTM

**Pull request merged**